devices.js

```diff
@@ -15184,6 +15184,26 @@ const devices = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        fingerprint: [
+            {modelID: 'TS0211', manufacturerName: '_TZ1800_ladpngdx'},
+        ],
+        model: 'HG06668',
+        vendor: 'Lidl',
+        description: 'Silvercrest Smart Wireless Door Bell',
+        fromZigbee: [
+            fz.battery,
+            fz.tuya_doorbell_button,
+            fz.ignore_basic_report,
+        ],
+        toZigbee: [],
+        meta: {
+            configureKey: 1,
+        },
+        exposes: [
+            e.battery(), e.action('pressed'), e.battery_low(), e.tamper(),
+        ],
+    },
```

Comment:
Regarding battery, try adding this:

```
        meta: {configureKey: 1},
        configure: async (device, coordinatorEndpoint, logger) => {
            const endpoint = device.getEndpoint(1);
            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
            await reporting.batteryPercentageRemaining(endpoint);
        },
```

Next trigger a configure via: https://www.zigbee2mqtt.io/information/mqtt_topics_and_message_structure.html#zigbee2mqttbridgerequestdeviceconfigure and check logs if OK.

Reply:
Ok. I got this in the logs:

`Jan 21 11:21:24 zigbee npm[7448]: Zigbee2MQTT:info  2021-01-21 11:21:24: MQTT publish: topic 'zigbee2mqtt/bridge/response/device/configure', payload '{"data":{"id":"Doorbell_Button_1"},"status":"ok"}'
Jan 21 11:21:24 zigbee npm[7448]: Zigbee2MQTT:error 2021-01-21 11:21:24: Failed to configure 'Doorbell_Button_1', attempt 1 (Error: Bind 0x842e14fffef94854/1 genPowerCfg from '0x00124b0021cbec97/1' failed (AREQ - ZDO - bindRsp after 10000ms)
Jan 21 11:21:24 zigbee npm[7448]:     at Timeout._onTimeout (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/utils/waitress.js:46:35)
Jan 21 11:21:24 zigbee npm[7448]:     at listOnTimeout (internal/timers.js:549:17)
Jan 21 11:21:24 zigbee npm[7448]:     at processTimers (internal/timers.js:492:7))
Jan 21 11:22:06 zigbee npm[7448]: Zigbee2MQTT:info  2021-01-21 11:22:06: Configuring 'Doorbell_Button_1'
Jan 21 11:22:09 zigbee npm[7448]: Zigbee2MQTT:info  2021-01-21 11:22:09: MQTT publish: topic 'zigbee2mqtt/Doorbell_Button_1', payload '{"battery":100,"battery_low":false,"last_seen":"2021-01-21T11:22:09.718Z","linkquality":126,"tamper":false}
Jan 21 11:22:09 zigbee npm[7448]: Zigbee2MQTT:info  2021-01-21 11:22:09: Successfully configured 'Doorbell_Button_1'
'

`

Reply:
Can you provide the data/database.db entry of this device?

Reply:
Of course:
```
{"id":35,"type":"EndDevice","ieeeAddr":"0x842e14fffef94854","nwkAddr":56068,"manufId":4619,"manufName":"_TZ1800_ladpngdx","powerSource":"Battery","modelId":"TS0211","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":1026,"inClusterList":[0,1,3,1280,2821],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"TS0211","manufacturerName":"_TZ1800_ladpngdx","powerSource":3,"zclVersion":3,"appVersion":69,"stackVersion":2,"hwVersion":16,"dateCode":"2020.8.4"}},"ssIasZone":{"attributes":{"iasCieAddr":"0x00124b0021cbec97","zoneState":1}},"genPowerCfg":{"attributes":{"batteryPercentageRemaining":200}}},"binds":[{"cluster":1,"type":"endpoint","deviceIeeeAddress":"0x00124b0021cbec97","endpointID":1}],"configuredReportings":[{"cluster":1,"attrId":33,"minRepIntval":3600,"maxRepIntval":62000,"repChange":0}],"meta":{}}},"appVersion":69,"stackVersion":2,"hwVersion":16,"dateCode":"2020.8.4","zclVersion":3,"interviewCompleted":true,"meta":{"configured":1},"lastSeen":1611244144515}
```

Reply:
Did you wakeup the device right before sending the command? (by clicking on it). Looks that the device supports the `genPowerCfg` (= 1) since it is in the `inClusterList`

Reply:
I don't remember but it kinda worked.  Triggered another configure now and got no error.
Also I deleted and reconnected it. No errors here too. But now I now longer get the "battery"=100 in the payloads

Reply:
It takes some time to report

Reply:
Restarting zigbee2mqtt fixed it. It now configured correctly

Reply:
```
Jan 22 18:29:48 zigbee npm[12588]: Zigbee2MQTT:info  2021-01-22 18:29:48: Configuring 'Doorbell_Button_1'
Jan 22 18:29:50 zigbee npm[12588]: Zigbee2MQTT:info  2021-01-22 18:29:50: MQTT publish: topic 'zigbee2mqtt/Doorbell_Button_1', payload '{"battery":100,"battery_low":false,"last_seen":"2021-01-22T18:29:50.767Z","linkquality":144,"tamper":false}'
Jan 22 18:29:50 zigbee npm[12588]: Zigbee2MQTT:info  2021-01-22 18:29:50: Successfully configured 'Doorbell_Button_1'
```

Reply:
Okay. I will add that to the PR then. Should I remove the battery_low then? That was just a guess based on other tuya devices.
Sorry for the comment spam.

Reply:
`battery_low` is fine (looks like its working since it false now)

Reply:
Okay I added everything